### DISCUSSION TITLE
Expose crowdAgent neighbours

### DISCUSSION THREAD

**Discussion by rob-myers** [isaac-mason/recast-navigation-js#443] [Ideas]
Hey, I'd like to access [dtCrowdAgent::neis](https://recastnav.com/structdtCrowdAgent.html#a924fad92f7bfcf396a3340d0e518bedf) i.e. the neighbours of a crowd agent.

For example, it is useful to know if another agent is in close proximity.

Can I open a PR?

**Comment by rob-myers** [comment]
Oops, this is already available:

```js
// neighbourIndex in [0...agent.raw.nneis - 1]
agent.raw.neis(neighbourIndex)

```

**Reply by isaac-mason** [reply]
Hey @rob-myers! It may be good to add a helper to the CrowdAgent class. eg a similar helper was added for `corners`

```ts
  corners(): Vector3[] {
    const points: Vector3[] = [];

    for (let i = 0; i < this.raw.ncorners; i++) {
      points.push({
        x: this.raw.get_cornerVerts(i * 3),
        y: this.raw.get_cornerVerts(i * 3 + 1),
        z: this.raw.get_cornerVerts(i * 3 + 2),
      });
    }

    return points;
  }
```

Feel free to open a PR for this :) otherwise I'll do this sometime.
